aiport-challenge-domain-models.md

```diff
@@ -30,13 +30,16 @@ I want to prevent asking the airport to let planes take-off which are not at the
 | ------- | ------------------------------ | ------------------------ | -------- |
 | Airport | airportCapacity @Integer       | landPlane(@Plane)        | @Void    |
 |         | planesAtAirport @Array[@Plane] | getCapacity()            | @Integer |
+|         |                                | checkPlaneValid(@Plane)  | @Error   |
```

Comment:
I spent a lot of time writing the Domain model initially before I even started to code, but when it came to development I found that different methods and properties were needed. Because the development was agile and test driven it allowed me to come back and change the model to reflect what I had found.